packages/caliper-tests-integration/scripts/run-tests.sh

```diff
@@ -52,6 +52,10 @@ elif [[ "${BENCHMARK}" == "fabric" ]]; then
         rc=$?
         exit $rc;
     fi
+elif [[ "${BENCHMARK}" == "ethereum" ]]; then
+    ${CALL_METHOD} benchmark run --caliper-benchconfig benchmark/simple/config-ethereum.yaml --caliper-networkconfig network/ethereum/1node-clique/ethereum.json --caliper-workspace ../caliper-samples/
```

Comment:
Since config-ethereum.yaml should be removed, don't forget to change the benchconfig filename here.